students/jungmin/westagram/users/views.py

```diff
@@ -27,10 +28,14 @@ def post(self, request):
             if User.objects.filter(email=email).exists():
                 return JsonResponse({'message': 'EMAIL_MUST_BE_UNIQUE'}, status=400)
 
+            encoded_password = password.encode('utf-8')
+            hashed_password  = bcrypt.hashpw(encoded_password, bcrypt.gensalt())
+            decoded_password = hashed_password.decode('utf-8')
```

Comment:
너무 잘 작성해주셨습니다! 👍

추가적으로 다음과 같이 작성할 수도 있을 것 같습니다.

```suggestion
            hashed_password  = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Reply:
다시 쓰는 변수가 아니라서 이렇게 하는게 맞겠네요!(맞죠..?)
감사합니다

Reply:
변수에 할당하지 않고 쓰는 방법과 이유까지!! 배우고 갑니다 👍👍